### PR TITLE
[RANS] Cleanup

### PR DESCRIPTION
**Pull request by sunethwarna**
This does cleanup and fixes warnings from clang.

Following bugs were also fixed:

1. Changes due to change in assign_scalar_variable_process
2. Moving two tests to small tests suite

One MPI test is disabled, for some reason the solver doesnt reach the expected level of residual.

### CODE REVIEW

File: applications/RANSApplication/tests/test_RANSApplication_mpi.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def AssembleTestSuites():
     ### Nightly MPI tests ######################################################
     nightlyMPISuite = suites['mpi_nightly']
     nightlyMPISuite.addTest(EvmKEpsilonTest('testChannelFlowKEpsilonTransientMPI'))
-    nightlyMPISuite.addTest(EvmKEpsilonTest('testChannelFlowKEpsilonSteadyMPI'))
+    # nightlyMPISuite.addTest(EvmKEpsilonTest('testChannelFlowKEpsilonSteadyMPI'))
```
Comment (philbucher) [inline applications/RANSApplication/tests/test_RANSApplication_mpi.py:42]: intended?

**Review by philbucher — COMMENTED** [review]
nice

why do you replace `typedef` with `using`?

File: applications/RANSApplication/tests/test_RANSApplication_mpi.py (hunk 1 above)
Comment (sunethwarna) [reply applications/RANSApplication/tests/test_RANSApplication_mpi.py:42]: yeah... this test is failing due to linear solver failing to reach the desired residual level. (Steady solutions are really hard to get convergence, I am at the moment working on a solution to make this better by stabilization). For the time being, i removed it.

**Comment by sunethwarna** [comment]
i changed "typedef" to "using" to be consistent, and looks more lceaner since it follows same and easy for eyes since it is from left to right as everyewhere else....

**Review by philbucher — APPROVED** [review]

**Pull request merged**